Scripts/Python/xHighLevelStarTrekDoor.py

```diff
@@ -144,219 +129,85 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
-                PtDebugPrint("xHighLevelStarTrekDoor: Notifying Clients to play Open Door Responder")
-
-            elif self.DoorState == doorSDLstates['closing']:
-                self.SendNote("respCloseDoor;0")
-                PtDebugPrint("xHighLevelStarTrekDoor: Notifying Clients to play Close Door Responder")
-
-            elif self.DoorState == doorSDLstates['open']:
-                PtDebugPrint("xHighLevelStarTrekDoor: Updating Age SDL to Open")
-                ageSDL[strDoorClosedVar.value] = (0,)
-
-            elif self.DoorState == doorSDLstates['closed']:
-                PtDebugPrint("xHighLevelStarTrekDoor: Updating Age SDL to Closed")
-                ageSDL[strDoorClosedVar.value] = (1,)
-
-    ##########################################
-    # This is a cheap hack to attempt to get 
-    # the doors reset, should the age owner 
-    # screw up and not update the door states
-    ##########################################
-    def OnTimer(self,TimerID):
-        if self.sceneobject.isLocallyOwned():
-            PtDebugPrint("xHighLevelStarTrekDoor: Timer Came Back")
-            if self.DoorState == doorSDLstates['opentoclose']:
-                self.UpdateDoorState(doorSDLstates['closing'])
-
-            elif self.DoorState == doorSDLstates['movingopen'] or self.DoorState == doorSDLstates['opening']:
-                self.UpdateDoorState(doorSDLstates['open'])
-
-            elif self.DoorState == doorSDLstates['closetoopen']:
-                self.UpdateDoorState(doorSDLstates['opening'])
-
-            elif self.DoorState == doorSDLstates['movingclosed'] or self.DoorState == doorSDLstates['closing']:
-                self.UpdateDoorState(doorSDLstates['closed'])
+                return
+            if int(playerID) == int(localPlayerID):
+                if self.DoorState == doorSDLstates['opening']:
+                    ageSDL[strDoorClosedVar.value] = (0,)
 
-    def ExecCode(self,code):
-        try:
-            chunks = code.split(';')
-            tag = chunks[0];
-            fastForward = int(chunks[1])
-            if tag == "respOpenDoor":
-                if fastForward == 1:
-                    respOpenDoor.run(self.key,fastforward=1)
-                else:
-                    respOpenDoor.run(self.key,netPropagate=0)
-            elif tag == "respCloseDoor":
-                if fastForward == 1:
-                    respCloseDoor.run(self.key,fastforward=1)
-                else:
-                    respCloseDoor.run(self.key,netPropagate=0)
-            else:
-                PtDebugPrint("xHighLevelStarTrekDoor.ExecCode(): ERROR! Invalid tag '%s'." % (tag))
-                self.DoorStack.pop(0)
-        except:
-            PtDebugPrint("xStandardDoor.ExecCode(): ERROR! Invalid code '%s'." % (code))
-            self.DoorStack.pop(0)
+                elif self.DoorState == doorSDLstates['closing']:
+                    ageSDL[strDoorClosedVar.value] = (1,)
```

Comment:
You need to remove the `sendToClients` bit on lines 82-83 or there will be a storm of SDL updates from every client in the Age.